.github/workflows/build-containers.yml

```diff
@@ -0,0 +1,42 @@
+name: Container Builds
+
+on:
+  push:
+    branches:
+    - main
+    paths:
+    - .github/workflows/build-containers.yml
+    - src/core/*
+    - src/platform/*
+    - src/perf/*
```

Comment:
Please include `bin` and `inc`. Why include `perf`? What do you think about `tools` for the sample?